homeworks/olha.tymchyshyn_olyatmchshn/5-logic/homework.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Perform a mathematical operation on two numbers. Return result or error.
+ * @param firstValue {string} with first number
+ * @param secondValue {string} with second number
+ * @param operation {string} with operation symbol
+ * @returns {string|number} result of calculation or error message:
+ * - if firstValue or secondValue is not a number, return 'Enter a number'
+ * - if operation is not '+', '-', '*', '/', return 'Choose a valid operation'
+ * - if the result is more than 100, return 'Result is too big'
+ * - else return number of result: 1 '+' 2 returns 3
+ *
+ */
+function calculate(firstValue, secondValue, operation) {
```

Comment:
Hey, @olyatmchshn, for this case should be an error 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/d12cfe01-19bd-4d1c-b40b-6b0194ebdf99)